### ISSUE TITLE
User is unable to delete first link in a collection

### ISSUE THREAD

**Issue by calisio** [UnlockedLabs/UnlockEdv2#467]
**Describe the bug**
When a user attempts to delete the first link in a collection, they are unable to do so.

**To Reproduce**

1. Go to Resource Management page on admin view
2. Click on a collection to modify
3. Attempt to delete the first link in the collection
4. See error - cannot successfully do this.

**Expected behavior**
You should be able to delete the first link. Also, if there is a problem deleting a link (or a collection for that matter), a toast should appear telling the user there is an error.

**Screenshots**

https://github.com/user-attachments/assets/9e19531e-318b-496a-8ed0-14b7a49eeed1

**Desktop (please complete the following information):**
Mac OS Version 14.5
Browser Chrome Version 130.0.6723.70

**Additional context**

**Comment by Arghya721** [comment]
@calisio there is an issue with `ResourcesManagement.tsx` where we are setting `activeLinkToDelete` to 0 when trying to delete the first link in a collection. Now in the check at line 686 
```
onSuccess={() =>
                            activeLinkToDelete &&
                            deleteLink(collection, activeLinkToDelete)
                        }
```
`activeLinkToDelete` is treated as false (since it is set to zero) hence it is not being deleted.

**Comment by jtucholski** [comment]
Closing this. I haven't been able to reproduce it and I think that it was addressed in #470 (like mentioned up above).
